### PR TITLE
add: date filters & max and min filter

### PR DESCRIPTION
**Pull request by SantiCervan**

### CODE REVIEW

File: app/components/SalesChart.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      labels: months,
+      datasets: [
+        {
+          label: `Ventas ${year}`,
+          data: newData,
+          backgroundColor: 'rgba(53, 162, 235, 0.5)',
+        },
+      ],
+    });
+  }
```
Comment (martinsd94) [inline app/components/SalesChart.tsx]: this can be updated into a custom webhook and reuse it

**Comment by martinsd94** [comment]
lets add .nvmrc file

**Comment by martinsd94** [comment]
also you can add prettier, eslint rules and pre-commit and pre-push hooks with husky

**Comment by martinsd94** [comment]
And the Cypress tests we discussed earlier

**Comment by martinsd94** [comment]
The sales data is refreshed every time a filter is updated. I believe it would better to update the sales data only when the "Generar Nuevas Ventas" button is clicked, WDYT?

BTW, filter styles look great now!

File: app/components/SalesChart.tsx (hunk 1 above)
Comment (SantiCervan) [reply app/components/SalesChart.tsx]: Resolved in commit 59b1a6e

**Comment by SantiCervan** [comment]
Resolved in [40e2137]

> lets add .nvmrc file

**Comment by SantiCervan** [comment]
> also you can add prettier, eslint rules and pre-commit and pre-push hooks with husky

Solved in [d399b33]

**Review by martinsd94 — APPROVED** [review]

**Pull request merged**